### ISSUE TITLE
Not centered

### ISSUE THREAD

**Issue by tablatronix** [remcoder/OctoPrint-GCodeThumbs#8]
Thumbs seem to show with a ton of padding to the left

![Screen Shot 2020-09-02 at 9 34 00 AM](https://user-images.githubusercontent.com/807787/91997407-ace92e80-ecff-11ea-977c-05efd4510a2b.png)

**Comment by remcoder** [comment]
Hi @tablatronix, which version are you using? I fixed an issue like this in v0.2.2 in May. You want want to try the latest version if you haven't already.

If you're already on the latest version then for me to inspect the issue I'll need one of your gcode files because different slicers output different flavors of gcode. You can use [my dropbox](https://www.dropbox.com/request/3eyeIzMrj2o7btJ3ybLV) to upload a gcode file.

**Comment by tablatronix** [comment]
I tried a few versions, and 0.2.2, I don't remember this before though, maybe i am mistaken though

These are coming out of cura, so maybe something there changed let me upload some old gcodes and see

**Comment by tablatronix** [comment]
ok uploaded some 
All files do it, might not be gcode.. I can debug

**Comment by remcoder** [comment]
thanks. I'll try to take a look this weekend

**Comment by surger13** [comment]
i have the same problem. the thumbnail shows only a lower left section.

many greetings

**Comment by remcoder** [comment]
Sorry but I haven't found the time to look into this. If anyone wants to debug and send in a PR I'll be happy to review and merge it but I'm not actively working on this anymore.

**Comment by tablatronix** [comment]
Is this canvas? Where do I start ? lol I will look

**Comment by remcoder** [comment]
@tablatronix yes canvas. Let me know if you have specific questions. I'll help you out.
